e2e/not-found.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
```

Comment:
```suggestion
import { test, expect } from '@playwright/test';
import translations from '../client/i18n/locales/english/translations.json';
```

We don't need a global `page` variable because...